app/js/actions/feedbackify.js

```diff
@@ -0,0 +1,5 @@
+import * as actionTypes from '../constants/action_types'
+
+export function openFeedbackDialog () {
+  return { type: actionTypes.OPEN_FEEDBACK_DIALOG }
+}
```

Comment:
This file is not needed, as the action doesn't modify the state of the app, just a global variable from the third party plugin.